sdk/src/main/res/drawable/kinecosystem_ic_arrow_right.xml

```diff
@@ -0,0 +1,10 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="24dp"
+    android:height="24dp"
+    android:viewportWidth="24"
+    android:viewportHeight="24">
+  <path
```

Comment:
are you sue vectors supported ?? did you check on device with minimum sdk

Reply:
yup works check on API 19